### DISCUSSION TITLE
Where do I find multicore test cases.

### DISCUSSION THREAD

**Discussion by HirunaVishwamith** [gem5/gem5#1435] [Q&A]
I'm trying to find some multicore testcases (C or assembly) to run on my RISC-V emulator. I hope there are test cases available here, bases on this paper "Simulating Multi-Core RISC-V Systems in gem5". If anyone know where to find these test cases pleas let me know.

Thank you.

**Comment by BobbyRBruce** [comment]
Why don't you use something like the NAS Parallel Benchmark suite for your work? Or you could write your own multicore C programs using OpenMP.
